client/lib/domains/utils.js

```diff
@@ -18,3 +24,20 @@ export function getDomainType( domainFromApi ) {
 
 	return domainTypes.MAPPED;
 }
+
+function getDomainNameFromReceiptOrCart( receipt, cart ) {
+	if ( receipt && isEmpty( receipt.failed_purchases ) ) {
+		return map( values( receipt.purchases )[ 0 ], 'meta' )[ 0 ];
+	}
+
+	if ( cartItems.hasDomainRegistration( cart ) ) {
+		return cartItems.getDomainRegistrations( cart )[ 0 ].meta;
+	}
+
+	return null;
+}
+
+export {
```

Comment:
Yeah, looked for that, but liked this more. Easier to look at the bottom for exported, if there are helpers.